etcdserver/v3_server.go

```diff
@@ -782,16 +786,26 @@ func (s *EtcdServer) linearizableReadLoop() {
 		if !done {
 			continue
 		}
+		trace.Step("read index received")
 
-		if ai := s.getAppliedIndex(); ai < rs.Index {
+		index := rs.Index
+		trace.AddField(traceutil.Field{Key: "readStateIndex", Value: index})
+
+		ai := s.getAppliedIndex()
+		trace.AddField(traceutil.Field{Key: "appliedIndex", Value: strconv.FormatUint(ai, 10)})
```

Comment:
Just a minor comment, why do we need to use `strconv.FormatUint` here, but use the original `uint64` value (line 792) above?

I understand you just  backport the PR, so this comment should be added for the original PR.

Reply:
I suggest to make them consistent, and update it in the main branch as well.

Reply:
I agree. logging should also be consistent with metric `proposalsApplied.Set(float64(ep.appliedi))` and etcdctl endpoint status output.

Reply:
This is just a backporting PR, so it might be better to merge it for now, and address the comment in separate PR(s).

Are you updating this PR? If not, then let me merge this PR firstly. If you are in progress of updating this PR, then feel free to continue to do it.